packages/react-components/src/Checkbox.tsx

```diff
@@ -48,6 +51,12 @@ export default React.memo(styled(Checkbox)`
     display: inline-block;
     margin: 0 0.5rem;
     opacity: 1;
+    cursor: pointer;
+    -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+    -moz-user-select: none; /* Old versions of Firefox */
+    -ms-user-select: none; /* Internet Explorer/Edge */
```

Comment:
Not against this, but since we are targetting newer browsers, it really should not be required.